change/@azure-msal-browser-883260a8-b8e9-4d81-a005-d5eb07364fbe.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

```suggestion
  "type": "patch",
```

Don't consider this worthy of a minor, as it isnt introducing new behavior.